aten/src/ATen/native/transformers/attention.cpp

```diff
@@ -680,10 +684,15 @@ Tensor scaled_dot_product_attention(
         auto out_lse_softmax = at::_scaled_dot_product_flash_attention(
             query_padded, key_padded, value_padded, dropout_p, is_causal, false /*return_debug_mask*/, og_scale.as_float_unchecked());
         return post_process_flash_output(std::get<0>(out_lse_softmax), og_size);
-      }
+      } else if (query_.device().type() == DeviceType::PrivateUse1) {
```

Comment:
This doesnt look right to me

It should now just be 1 more case switch entry


You will need to add the overridable backend
``` cpp
   case sdp::SDPBackend::overridable:
      return std::get<0>(at::_scaled_dot_product_attention_overridable(
         ...));```
         
         
         
     Private use authors would thsu register a dispatch to the stub and have it return the overrridable backend
     
     by default they would be routed to the math backend

Reply:
@drisspg  Could you please help review the PR again? Thanks!

Reply:
If you have any further questions, feel free to bring them up. @drisspg